hugo/README.md

```diff
@@ -67,6 +67,16 @@ If all above is done and the icon is added in the right folder you can run the f
 npm run icons
 ```
 
+### Algolia
+
+If you want to test Algolia local, you can add the environoment variables from Algolia to the environment config:
+
+```
+cp .env.sample .env
```

Comment:
Not everyone uses `.env` files - I would simply document what needs to be done to run Algolia locally